internal/pkg/skuba/addons/cilium.go

```diff
@@ -267,19 +267,27 @@ spec:
             command:
             - cilium
             - status
+            - --brief
           # The initial delay for the liveness probe is intentionally large to
           # avoid an endless kill & restart cycle if in the event that the initial
           # bootstrapping takes longer than expected.
           initialDelaySeconds: 120
           failureThreshold: 10
-          periodSeconds: 10
+          initialDelaySeconds: 5
```

Comment:
initialDelaySeconds for livenessProbe is still '120' and only for 'readinessProbe' it is 5.

Reply:
Otherwise it looks good to me.

Reply:
Ops... copy-paste issue. Sorry